ubuntu_server_triage.py

```diff
@@ -147,10 +181,18 @@ def main(start, end=None):
                         '(e.g. 2016-07-31)')
     PARSER.add_argument('-d', '--debug', action='store_true',
                         help='debug output')
+    PARSER.add_argument('-o', '--open', action='store_true',
+                        help='open in web browser')
+    PARSER.add_argument('-n', '--lpname', default='ubuntu-server',
+                        help='specify the launchpad name to search for')
+    PARSER.add_argument('-b', '--bugsubscriber', action='store_true',
+                        help=('filter name as bug subscriber (default would '
+                              'be structural subscriber'))
```

Comment:
Well that is the field name in the search you coded yourself :-)
It is the Team or Person that is the default owner or responsible for a packet.
That seems to be the LP Term for it.